.github/workflows/editorconfig.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Why is this required now?

Reply:
Didn't get that in mine - https://github.com/eyJhb/nixpkgs/pull/1/files

Reply:
It was added by @zowoq. Explanation above.